src/vue/forms/CreateIssuanceForm.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <form
+    v-if="isShown"
```

Comment:
Why do we need isShown property here? We already have the `isShown` prop on drawer, isn't it enough?

Reply:
Resolved, removed